state/watcher.go

```diff
@@ -1619,6 +1619,9 @@ func (w *unitsWatcher) loop(coll, id string) error {
 			return tomb.ErrDying
 		case <-rootCh:
 			changes, err = w.update(changes)
+			if err != nil {
```

Comment:
Again, it looked like the original code was written to ignore the error intentionally.

Reply:
So this should be `change, _ = w.update(changes)`?